public/client/search.js

```diff
@@ -1,17 +1,95 @@
 // request button- activate modal
-function requestBook(bookId, pointCost, title, userPoints){
+function requestBook(bookId, pointCost, title, userPoints, condition, date){
+    var condition = condition.split(',');
+    var date = date.split(',');
+    var modalBody = document.getElementById('modalBody');
+    var selector = document.getElementsByTagName('select')[0];
+    if(selector != undefined){
+        var div1 = document.getElementById("div1")
+        modalBody.removeChild(div1);
+    }
     if(pointCost > userPoints){
         $("#pointsModal").modal("show");
     }
-    else{
-        var confirmButton = confirm("Are you sure you want " + title + "?");
-        if(confirmButton){
-            request(bookId, pointCost, title);
+    else if(bookId.length > 1){
+        modalBody.innerHTML = `Are you sure you want this book?: <br> <b>"${title}"</b><br>`
```

Comment:
Same with this. There was a previous PR that had this modal created like this and a request was made to stay in line with the rest of the application and create the modal with bootstrap on the page and using the standard confirm message window. With this pull request, you have undone all of Glenn's refactoring.